packages/styles/popover.css

```diff
@@ -0,0 +1,117 @@
+:root {
+  --popover-background-color: var(--gray-10);
+  --popover-border-color: var(--gray-90);
+  --popover-text-color: var(--gray-90);
+  --popover-box-shadow: rgba(0, 0, 0, 0.35) 0px 0px 10px 0px;
+  --popover-button-height: 24px;
+  --popover-alert-max-width: 192px;
+  --popover-alert-min-height: 156px;
```

Comment:
We generally avoid fixed width/height in px since that can cause issues if users are resizing only font sizes. These should be converted to `rem` instead.